### ISSUE TITLE
how to reproduce ResNet-152 + Bi-GRU on COCO experiment

### ISSUE THREAD

**Issue by Asakinevergup** [kdwonn/DivE#11]
Hello! I'm reproducing your experiments. I followed your train_eval_coco.sh script and obtained the same results. However, when I attempted to reproduce the experiment with ResNet-152 + Bi-GRU, the results were not as expected. Could you please tell me how you configured it? Thank you very much!
PS: These are the parameters I used during training.
python3 train.py \
--data_name coco --cnn_type resnet152 --wemb_type glove \
--margin 0.2 --max_violation --img_num_embeds 4 --txt_num_embeds 4 \
--img_attention --txt_attention --img_finetune --txt_finetune \
--mmd_weight 0.01 --unif_weight 0.01 \
--batch_size 200 --warm_epoch 0 --num_epochs 80 \
--optimizer adamw --lr_scheduler cosine --lr_step_size 30 --lr_step_gamma 0.1 \
--warm_img --finetune_lr_lower 1 \
--lr 1e-5 --txt_lr_scale 10 --img_pie_lr_scale 10 --txt_pie_lr_scale 10 \
--eval_on_gpu --sync_bn --amp \
--loss smooth_chamfer --eval_similarity smooth_chamfer --temperature 16 \
--txt_pooling rnn --arch slot --txt_attention_input wemb \
--spm_img_pos_enc_type sine --spm_txt_pos_enc_type sine \
--spm_1x1 --spm_residual --spm_residual_norm --spm_residual_activation none \
--spm_activation gelu \
--spm_ff_mult 4 --spm_last_ln \
--img_res_pool max --img_res_first_fc \
--spm_input_dim 1024 --spm_query_dim 1024 \
--spm_depth 4 --spm_weight_sharing \
--remark coco \
--res_only_norm --img_1x1_dropout 0.1 --spm_pre_norm \
--gpo_1x1 --gpo_rnn \
--weight_decay 1e-4 --grad_clip 1 --lr_warmup -1 --unif_residual \
--workers 4 --dropout 0.1 --caption_drop_prob 0.2 --butd_drop_prob 0.2

**Comment by WangHNoob** [comment]
Hello! Have you tackled the problem?

**Comment by Asakinevergup** [comment]
Unfortunately, no  but  u can  try more times, good luck!
